src/status_im2/contexts/onboarding/common/intro/style.cljs

```diff
@@ -0,0 +1,60 @@
+(ns status-im2.contexts.onboarding.common.intro.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def progress-bar-container
+  {:margin-vertical 16
+   :flex-direction  :row})
+
+(defn get-color
+  [index position]
+  (if (= index position) colors/white colors/white-opa-10))
+
+(defn progress-bar-item
+  [index position end?]
+  {:height       2
+   :flex         1
+   :border-width 1
+   :border-color (get-color index position)
+   :margin-right (if end? 0 8)})
+
+(def carousel
+  {:height             92
+   ;; (padding-top) This insets issue needs a consistent implementation across all screens.
```

Comment:
Please can you elaborate?

Reply:
I had to pad this page, I would rather do it from the navigation rather than on the specific page. I assume it will happen for all onboarding flows.